Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -1049,9 +1059,14 @@ GRefPtr<GstElement> GStreamerMediaEndpoint::requestAuxiliarySender(GstWebRTCDTLS
         return nullptr;
     }
 
-    auto estimator = adoptGRef(makeGStreamerElement("rtpgccbwe", nullptr));
+    // Don't use makeGStreamerElement() here because it would be called mutiple times and emit an
+    // error every single time if the element is not found.
+    auto estimator = adoptGRef(gst_element_factory_make("rtpgccbwe", nullptr));
     if (!estimator) {
-        GST_WARNING_OBJECT(m_pipeline.get(), "gst-plugins-rs is not installed, RTP bandwidth estimation now disabled");
+        static std::once_flag onceFlag;
+        std::call_once(onceFlag, [] {
+            WTFLogAlways("gst-plugins-rs is not installed, RTP bandwidth estimation now disabled");
```

Comment:
Maybe we can keep the gst warning as well

Reply:
One is enough imho